server/tests/src/main/scala/sttp/tapir/server/tests/ServerStaticContentTests.scala

```diff
@@ -494,7 +494,10 @@ class ServerStaticContentTests[F[_], OPTIONS, ROUTE](
       serveRoute(resourcesGetServerEndpoint[F](emptyInput)(classOf[ServerStaticContentTests[F, OPTIONS, ROUTE]].getClassLoader, "test"))
         .use { port =>
           get(port, List("r1.txt")).map { r =>
-            r.contentLength shouldBe Some(10)
+            val headersToValues = r.headers.map(h => (h.name.toLowerCase, h.value))
```

Comment:
file has a known length so this shuoldn't use chunked encoding - the test was correct before

Reply:
reverted, updated code responsible for setting headers, so that if the length is provided it is taken from there.